### DISCUSSION TITLE
Ask to delete linked files/images from attachments folder when deleting a note

### DISCUSSION THREAD

**Discussion by vladstudio** [notaapp/talk#287] [Feature Requests]
When I'm deleting a note, I most likely want to also delete all files and images that this note links to (and that are not linked from any other notes). 

Maybe it's a good idea to add a checkbox saying "also remove X linked files" (if any) to the confirmation dialog? This one:

<img width="320" alt="image" src="https://user-images.githubusercontent.com/914320/183309381-68192e5b-0e0b-4a57-ac7d-1ac20a9893e2.png">

**Comment by astoilkov** [comment]
I like the idea. I also want to see a list of those files (especially for notes). Do you? I'm not sure I will be comfortable selecting the checkbox if I don't know which files will be deleted.

**Reply by vladstudio** [reply]
True! Because file list is unpredictable, I'd hide it behind the fold. There'd be an "expand" arrow I could click to review file paths, similar to:

<img width="266" alt="image" src="https://user-images.githubusercontent.com/914320/183376373-65a69df3-27e2-4c0c-806a-e214016ecc50.png">

**Reply by vladstudio** [reply]
OK the screenshot might be misleading, so I made a sketch:

<img width="706" alt="image" src="https://user-images.githubusercontent.com/914320/183380113-a5e4df16-b634-412b-a09c-ccc31e1d8dd1.png">

**Comment by Arbiter313** [comment]
On a similar topic: when you drag a note from the workspace to another location it creates a copy of it in that location, so it might be a good idea to have a similar prompt to make a copy any linked images as well?

**Reply by astoilkov** [reply]
Writing the idea down. Thanks!
